File: 1672-richest-customer-wealth/1672-richest-customer-wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    res = 0
    for(let i = 0 ; i < accounts.length ; i++)
        {
            let perincome = 0
            for(let j = 0 ; j < accounts[i].length ; j++) perincome += accounts[i][j]
            res = Math.max(res, perincome)
        }
    return res
};

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
       int maxx =  INT_MIN;
        for(int i = 0 ; i < accounts.size(); i++)maxx = max(maxx, accumulate(accounts[i].begin(), accounts[i].end(), 0));
        return maxx;
    }
};